torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -25,7 +25,6 @@
 )
 from torch.testing._internal.common_distributed import (
     skip_if_lt_x_gpu,
-    skip_if_no_peer_access,
```

Comment:
Was the `skip_if_no_peer_access` decorator added specifically for these tests? If so, we might want to remove it altogether.

Reply:
yep, let me remove that function